packages/url/package.json

```diff
@@ -19,7 +19,8 @@
 	"main": "build/index.js",
 	"module": "build-module/index.js",
 	"dependencies": {
-		"@babel/runtime": "^7.0.0-beta.52"
+		"@babel/runtime": "^7.0.0-beta.52",
+		"qs": "^6.5.2s"
```

Comment:
See the issue on this line? 😱 

We really need to do something to hold ourselves more accountable for published packages working as advertised. This line makes the packages largely unusable, or at least in my attempts to use `@wordpress/components` on [plnkr.co ](http://plnkr.co)

Reply:
I wonder how it worked on Calypso :)

Reply:
Hmm, maybe npm is resilient enough to be allowing the extra character? At least based on the resolved `qs` in `package-lock.json`.